types/tailwindcss/index.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for tailwindcss 2.0
+// Project: https://github.com/tailwindlabs/tailwindcss
+// Definitions by: Dolan Miu <https://github.com/dolanmiu>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 4.1
+
+import { TailwindConfig } from './tailwind-config';
```

Comment:
```suggestion
import type { TailwindConfig } from './tailwind-config';
```

Reply:
and also, why this cannot be in `lib/util/resolveConfig`?

Reply:
Wrote reasoning above, I will repeat:

It can't be lib/util/resolveConfig because it has to be:

import resolveConfig from 'tailwindcss/resolveConfig'
Is this what you mean?

Source:

https://tailwindcss.com/docs/configuration#referencing-in-java-script

Screen shot of documentation:

![Screenshot 2021-02-02 at 22 27 17](https://user-images.githubusercontent.com/2917613/106671086-d60b3500-65a5-11eb-879f-f9ecbad29b35.png)